test/infrastructure/directories/api.getUsersByIdV2.test.js

```diff
@@ -0,0 +1,94 @@
+jest.mock('login.dfe.async-retry');
+jest.mock('login.dfe.jwt-strategies');
+jest.mock('./../../../src/infrastructure/config', () => require('../../utils').configMockFactory({
+  directories: {
+    type: 'api',
+    service: {
+      url: 'http://directories.test',
+    },
+  },
+}));
+
+const { fetchApi } = require('login.dfe.async-retry');
+const jwtStrategy = require('login.dfe.jwt-strategies');
+const { getUsersByIdV2 } = require('../../../src/infrastructure/directories/api');
+
+const correlationId = 'abc123';
+const apiResponse = {
+  users: [],
+  numberOfPages: 1,
+};
+const userIds = ['user1', 'user2'];
+
+describe('when getting a page of users from directories api', () => {
+  beforeEach(() => {
+    fetchApi.mockReset();
+    fetchApi.mockImplementation(() => {
+      return apiResponse;
+    });
+
+    jwtStrategy.mockReset();
+    jwtStrategy.mockImplementation(() => {
+      return {
+        getBearerToken: jest.fn().mockReturnValue('token'),
+      };
+    })
+  });
+
+  it('then it calls directories api with Ids', async () => {
```

Comment:
This has a POST because the IDs are passed through the body, this check doesn't check for the IDs like the description states 😄